src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseServiceTest.java

```diff
@@ -265,7 +267,71 @@ void formatViewApplication_withAllData() {
                 + "|Case management order made by|Legal officer|\r\n"
                 + "|Request made by|Caseworker|\r\n"
                 + "|Full name|Mr Lee Gal Officer|\r\n"
-                + "|Sent to|Respondent|\r\n\r\n";
+                + "|Sent to|Respondent|\r\n\r\n\r\n";
+
+        assertThat(tseService.formatViewApplication(caseData, AUTH_TOKEN)).isEqualTo(expected);
+    }
+
+    @Test
+    void formatViewApplication_withAllDataAndDecision() {
```

Comment:
Fix the name of the previous method if it didn't have all data before. Also include multiple decisions to show it can handle it (have more than 2 with different dates and show they pick the latest 2)